modules/rgbd/src/colored_kinfu.cpp

```diff
@@ -172,13 +173,13 @@ ColoredKinFuImpl<MatType>::ColoredKinFuImpl(const Params &_params) :
                         params.tsdf_trunc_dist, params.tsdf_max_weight, params.truncateThreshold,
                         params.volumeDims[0], params.volumeDims[1], params.volumeDims[2]);
 
-    icp = FastICPOdometry::create(Mat(params.intr), params.icpDistThresh, params.icpAngleThresh,
-                                  params.bilateral_sigma_depth, params.bilateral_sigma_spatial, params.bilateral_kernel_size,
-                                  params.icpIterations, params.depthFactor, params.truncateThreshold);
+    OdometrySettings ods;
```

Comment:
What about other settings passed in `FastICP` constructor? Are they already set by default?

Reply:
yes